Makefile

```diff
@@ -287,10 +287,8 @@ ifneq ($(filter -DROCKSDB_LITE,$(OPT)),)
 endif
 
 ifeq ($(LIB_MODE),shared)
-ifneq ($(DEBUG_LEVEL),0)
-# So that debug binaries can find shared libraries in build location
-EXEC_LDFLAGS += -Wl,-rpath="$(PWD)"
-endif
+# So that binaries are executable from build location, in addition to install location
+EXEC_LDFLAGS += -Wl,-rpath='$$ORIGIN'
```

Comment:
This is failing to compile on MacOS:

CCLD     cache_bench
ld: unknown option: -rpath=$ORIGIN
clang: error: linker command failed with exit code 1 (use -v to see invocation)

If I specify -rpath . on the Mac (no =), it works.  I have not gotten it to work with ORIGIN however

Reply:
I wonder if this option should go into build_detect_platform as part of PLATFORM_LDFLAGS?

Reply:
Except it's EXEC_LDFLAGS (only for linking executables), so PLATFORM_EXEC_LDFLAGS?